Procfile.example

```diff
@@ -0,0 +1,48 @@
+# Web server workers
+# The $PORT variable is set automatically by foreman for particular workers
+# We tell foreman to start at 3000, it then increments by 1 per instance of a worker type
+# and 100 per worker type.
+#
+# So, the first ui worker will get PORT=3000 while the next will get PORT=3001
+# The first api worker will get PORT=3100 and the next will get PORT=3101
+
+# ui:        env PORT=$PORT bundle exec rails r lib/workers/bin/run_single_worker.rb MiqUiWorker
+# api:       env PORT=$PORT bundle exec rails r lib/workers/bin/run_single_worker.rb MiqWebServiceWorker
+# websocket: env PORT=$PORT bundle exec rails r lib/workers/bin/run_single_worker.rb MiqWebsocketWorker
```

Comment:
I can't seem to get past the login page without using rails r here.

Even when running all the workers (in case login was pinging the API or something)